pages/index.js

```diff
@@ -94,6 +92,7 @@ const CarouselHero = styled.div`
 `
 
 export default function Home() {
+
```

Comment:
I guess this newline is leftover. However, keep like this in this PR, after merging we will add a PR passing with `prettier`.